source/expressions.tex

```diff
@@ -1158,7 +1158,7 @@
 \grammarterm{literal}
 is a primary expression.
 Its type depends on its form\iref{lex.literal}.
-A string literal is an lvalue; all other literals are prvalues.
+A \grammarterm{string-literal} is an lvalue; all other literals are prvalues.
```

Comment:
Hmm. The existing wording here is just plain wrong. UDLs can have any value category.

Reply:
Applied a larger hammer.